clang/test/Sema/cheri/underalign2.c

```diff
@@ -1,26 +1,22 @@
-// Check that __attribute__((__aligned__)) aligns to sizeof(void* __capability) even for Cheri256
-// RUN: %cheri128_cc1 -target-cpu mips4 -mllvm -cheri-cap-table-abi=pcrel -std=c11 %s -verify -emit-llvm -o - | FileCheck '-D$CAP_SIZE=16' '-D$GLOBALS_AS=' %s
-// RUN: %cheri256_cc1 -target-cpu mips4 -mllvm -cheri-cap-table-abi=pcrel -std=c11 %s -verify -emit-llvm -o - | FileCheck '-D$CAP_SIZE=32' '-D$GLOBALS_AS=' %s
-// RUN: %cheri128_purecap_cc1 -target-cpu mips4 -mllvm -cheri-cap-table-abi=pcrel -std=c11 %s -verify -emit-llvm -o - | FileCheck '-D$CAP_SIZE=16' '-D$GLOBALS_AS=addrspace(200) ' %s
-// RUN: %cheri256_purecap_cc1 -target-cpu mips4 -mllvm -cheri-cap-table-abi=pcrel -std=c11 %s -verify -emit-llvm -o - | FileCheck '-D$CAP_SIZE=32' '-D$GLOBALS_AS=addrspace(200) ' %s
-
-struct no_caps {
-  int i;
-} __attribute__((__aligned__));
-struct no_caps a;
-// CHECK: @a = [[$GLOBALS_AS]]global %struct.no_caps zeroinitializer, align [[$CAP_SIZE]]
-
-struct with_cap {
-  int *__capability i;
-} __attribute__((__aligned__));
-struct with_cap b;
-// CHECK: @b = [[$GLOBALS_AS]]global %struct.with_cap zeroinitializer, align [[$CAP_SIZE]]
+// Check that __attribute__((__aligned__)) aligns to at least sizeof(void* __capability) even for Cheri256
```

Comment:
Ditto

Reply:
I kept that one since it checks that it uses max(sizeof(void*), 16) instead of just hardcoding 16. But I agree it's not particularly useful.